apiserver/modelconfig/modelconfig_test.go

```diff
@@ -48,6 +49,22 @@ func (s *modelconfigSuite) SetUpTest(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *modelconfigSuite) TestAccessDenied(c *gc.C) {
```

Comment:
This test isn't really part of this PR anymore now that the methods have been correctly added to the uniter facade.
Just delete the test IMO